tests/Homely.AspNetCore.Mvc.Helpers.Tests/JsonOptionsTests/AddJsonOptionsTests.cs

```diff
@@ -38,9 +38,13 @@ public AddJsonOptionsTests(TestFixture factory)
             // Arrange.
             var client = _factory.WithWebHostBuilder(builder =>
             {
+                var baseClassConverter = new BaseClassConverter();

```

Comment:
hang on.. so we _do_ need a custom converter now? i thought the only problem was we were declaring the property as the base class, instead of the concrete?

Reply:
Nope. we do not need a custom converter. I just wanted to test things out to make sure we can deserialize .. and while i'm doing this, check that custom converters work (refer to the ctor option in the `AddJsonOptions` class).